scripts/package/postinstall.sh

```diff
@@ -70,7 +71,7 @@ The collector's config file can be found here:
   /opt/observiq-otel-collector/config.yaml
 
 To view logs from the collector, run:
-  sudo journalctl -u observiq-otel-collector.service
+  sudo tail -F /opt/observiq-otel-collector/log/collector.log
```

Comment:
Do we need to update the mac or windows docs to reflect this also?

Reply:
Took a pass at docs; Updated install docs and where it was referenced in install scripts too.